lib/generators/allinson_flex/works_generator.rb

```diff
@@ -27,6 +28,25 @@ def generate_works
       say_status("info", "GENERATING #{work_type}", :blue)
       generate "hyrax:work #{work_type}", '-f'
     end
+
+  end
+
+  def add_additional_modules
+    # add additional modules to generated classes
+    # example: rails generate allinson_flex:works --include_module ModuleName
```

Comment:
this allows us to include `bulkrax_identifier` as a property to all models made with allinson flex by passing the module name in. (ref: https://gitlab.com/notch8/utk-hyku/-/merge_requests/93)

however, in order to keep it dynamic, we're allowing any module name to be passed in.

Reply:
# TODO allow multiple modules to be passed to this method